content/en/logs/log_configuration/log_optimizer.md

```diff
@@ -0,0 +1,87 @@
+
+Log Optimizer helps you identify log patterns that generate high volumes of repetitive or noisy data. Datadog analyzes your indexed logs and recommends actions, such as excluding, sampling, or converting logs to metrics, so you can optimize log volumes and focus on the most relevant information for troubleshooting and analysis.
+
+This feature builds on [Logging without Limits™][1] and complements tools such as [Exclusion Filters][2] and [Logs to Metrics][3].
+
+## How it works
+
+Datadog continuously reviews your **indexed** logs to find patterns that generate large or repetitive volumes of data. Once per day, Log Optimizer evaluates these patterns against Datadog best practices and identifies logs that may benefit from optimization.
+
+The Log Optimizer then suggests actions (such as excluding debug-level messages, sampling routine logs, or converting static messages to metrics) so you can reduce noise without losing visibility into important events.
+
+<div class="alert alert-danger">The Log Optimizer does not account for existing exclusion filters or logs-to-metrics conversions. Review your settings before applying new actions to avoid duplicates.</div>
+
+### What Datadog analyzes
+
+* **Indexed logs:** The analysis focuses on logs stored in your standard indexes.
+* **High-volume patterns:** Datadog detects patterns that make up a significant share of your total log volume.
+* **Message consistency and content:** Logs with repeated or low-variability messages are evaluated as potential candidates for optimization. For example, if log messages indicate successful operations (such as "process executed successfully"), Log Optimizer may recommend excluding those logs to reduce noise.
+
+### Recommended actions
+
+Each recommendation includes an explanation and a suggested action.
+
+| Recommendation | Description | Typical example |
+| :---- | :---- | :---- |
+| **Exclude** | Stop indexing logs that add noise and make it harder to focus on critical signals. | Debug-level messages or verbose system output. |
+| **Sample** | Lower the percentage of repetitive logs to reduce noise without losing visibility. | Logs with very little variability (where fields like timestamps or IDs might be the only change) |
+| **Convert to metric** | Replace repeated logs with a metric to track counts or trends over time. | Logs that always show the same message or status. |
+
+## Review and apply recommendations
+
+Navigate to the [**Log Optimizer**][4] page to view log patterns, sample messages, volume data, and plain-language explanations for each recommendation.
+
+To apply a recommendation:
+
+1. Click a recommendation to open the side panel.
+2. Click an action button (**Exclude Logs**, **Sample Logs**, or **Create Metric**).
+
+The change takes effect immediately in your configuration and the affected log pattern updates the next time Datadog runs the daily analysis.
```

Comment:
"the affected log pattern updates" ... Does this mean that the list of patterns on the Log Optimizer page will still list the identified recommendation, even though you took action on it, until the analysis re-runs the next day? Might want to be explicit about that because if it's still there it might feel like the exclusion filter or metric creation didn't work, and the person will try to do it again